Source/SPPreferencesUpgrade.m

```diff
@@ -80,11 +80,12 @@ void SPApplyRevisionChanges(void)
 	[prefs setObject:@YES forKey:@"doBundleUpdate"];
 
 	
-//	// FThis is how you add release notes and run specific migration steps
-//	if (recordedVersionNumber < 3922) {
-//		[importantUpdateNotes addObject:NSLocalizedString(@"The Custom Query \"Run\" and \"Run All\" button positions and their shortcuts have been swapped.", @"Short important release note for swap of custom query buttons")];
-//		[importantUpdateNotes addObject:NSLocalizedString(@"We've changed Sequel Ace's digital signature for GateKeeper compatibility; you'll have to allow access to your passwords again.", @"Short important release note for why password prompts may occur")];
-//	}
+	
+	// This is how you add release notes and run specific migration steps
+	if (recordedVersionNumber < 3922) {
```

Comment:
@jamesstout probably want to set the recordedVersionNumberComparison to 1 more than the current build number

Reply:
as in

`(recordedVersionNumber <2062) {`

?

Reply:
or 
`(recordedVersionNumber <currentVersionNumber+1) {`

Reply:
`(recordedVersionNumber <2061) {`
I'm silly, needs to be less than the previous build number + 1 (current build number) or it would show every time the app was opened

Reply:
`if (recordedVersionNumber < currentVersionNumber) {`

seems to work.

Reply:
Sorta, the issue with that is it will show for _every_ update. Even future ones. We really only want to show the specific update message when a user is going from a version older than 2061 to a version greater than 2061. We wouldn't want to show it to a user going from 2065 to 2067 because it's no longer applicable for them

Reply:
Just so I'm clear (it was 5am when I read this) .. I hard code to 2061.

`(recordedVersionNumber <2061) {`

@Kaspik @gboudreau if you're around. Brain not wokring....

Reply:
If it will be `recordedVersionNumber < currentVersionNumber`, it will be shown every release. If it will be vs hardcoded number, it will be shown only in the specific release.